src/commands/dev/index.js

```diff
@@ -175,11 +174,8 @@ const handleLiveTunnel = async ({ flags, site, api, settings, log }) => {
   }
 }
 
-const BANNER_LENGTH = 70
-
 const printBanner = ({ url, log }) => {
-  // boxen doesnt support text wrapping yet https://github.com/sindresorhus/boxen/issues/16
```

Comment:
This is [supported natively](https://togithub.com/sindresorhus/boxen/issues/46) and will wrap the content based on the size of the terminal instead of a hardcoded fixed value.